modules/fswatch/paramscreator.go

```diff
@@ -0,0 +1,22 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor: Aaron Meihm ameihm@mozilla.com [:alm]
+
+package fswatch /* import "mig.ninja/mig/modules/fswatch" */
+
+import (
+	"fmt"
+)
+
+func printHelp(isCmd bool) {
+	fmt.Printf(`Query parameters
+----------------
+This module has no parameters.
```

Comment:
Should it even implement these interfaces? `ParamsParser` and `ParamsCreator` are really for command line tools, and I don't see any use of those here.